server/index.js

```diff
@@ -1,10 +1,105 @@
 // Entry point of express server
 const express = require('express');
+const cors = require('cors');
+const { Pictures, ReviewPhotos } = require('../mongoDB/index');
+
 const app = express();
 const port = 3000;
 
-app.use(express.static(__dirname + '/../public'))
+app.use(cors());
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use(express.static(`${__dirname}/../public`));
+
+// GET request for photos corresponding to a item
+app.get('/pictures/itemId/:itemId', (req, res) => {
+  const { itemId } = req.params;
+  Pictures.find({ item_id: itemId })
+    .then((item) => {
+      const {
+        // eslint-disable-next-line camelcase
+        item_id, store_id, item_pictures, seller_picture, store_picture,
+      } = item[0];
+      const response = {
+        item_id,
+        store_id,
+        item_pictures,
+        seller_picture,
+        store_picture,
+      };
+      res.status(200).json(response);
+    })
+    .catch((err) => {
+      res.status(400).send('Error:', err);
+    });
+});
+
+// GET request for a list of all items that belong to a store
+app.get('/pictures/storeId/:storeId', (req, res) => {
+  const { storeId } = req.params;
+  Pictures.find({ store_id: storeId })
+    .then((items) => {
+      const response = items.map((item) => {
+        const {
+          // eslint-disable-next-line camelcase
+          item_id, store_id, item_pictures, seller_picture, store_picture,
+        } = item;
+        return ({
+          item_id, store_id, item_pictures, seller_picture, store_picture,
+        });
+      });
+      res.status(200).json(response);
+    })
+    .catch((err) => {
+      res.status(400).send('Error:', err);
+    });
+});
+
+// Batch Post Request: returns an array of photos corresponding to incoming itemIDs
+app.post('/pictures/batch/itemIds', (req, res) => {
+  const { itemIds } = req.body;
+  Pictures
+    .find()
+    .where('item_id')
+    .in(itemIds)
+    .then((items) => {
+      const response = items.map((item) => {
+        const {
+          // eslint-disable-next-line camelcase
+          item_id, store_id, item_pictures, seller_picture, store_picture,
+        } = item;
+        return ({
+          item_id, store_id, item_pictures, seller_picture, store_picture,
+        });
+      });
+      res.status(200).json(response);
+    })
+    .catch((err) => {
+      res.status(400).send('Error:', err);
+    });
+});
+
+// Batch Post Request: return an array of review photos corresponding to incoming ids
+app.post('/reviewPhotos/batch/itemIds', (req, res) => {
```

Comment:
Same here isn't this a Get? Pretty sure if you're only retrieving data it should be a get unless you're saving the searches for some reason.